lib/rucio/core/rule.py

```diff
@@ -1015,6 +1015,14 @@ def list_rules(
                 value = DIDType(value)
             elif key == 'grouping' and isinstance(value, str):
                 value = RuleGrouping(value)
+            elif key == 'name':
+                # First check if it is a real DID or a wildcard
+                if "*" in value:
+                    # Get the matching dids
+                    value = value.replace('*', '%')
+                    stmt = stmt.where(models.ReplicationRule.name.like(value))
```

Comment:
Should we add a `LIMIT` to wildcard queries to reduce excessive resource (like CPU memory, or I/O) consumption for an extended time leading to slow performance or system slowdowns, e.g. 

```suggestion
                    stmt = stmt.where(models.ReplicationRule.name.like(value))
                    stmt = stmt.limit(1000)  # Prevent runaway queries
```